input/fsh/plandefinition/CAREANDTREATMENT.CLINICAL.VISIT.fsh

```diff
@@ -0,0 +1,102 @@
+Alias: $plan-definition-type = http://terminology.hl7.org/CodeSystem/plan-definition-type
+
+Instance: CareAndTreatmentClinicalVisit-PlanDefinition
+InstanceOf: PlanDefinition
+Usage: #definition
+* name = "PlanDefinition_Care and treatment clinical visit"
+* title = "PlanDefinition - Care and treatment clinical visit"
+* type = $plan-definition-type#workflow-definition "Workflow Definition"
+* status = #active
+* experimental = true
+* publisher = "World Health Organization (WHO)"
+* description = "To initiate ART and to provide HIV care, treatment and integrated health services"
+* action.title = "Care and treatment clinical visit business process"
+* action.description = " To initiate ART and to provide HIV care, treatment and integrated health services"
+* action.participant[0].type = #patient
+* action.participant[=].role.coding.code = #client
+* action.participant[+].type = #practitioner
+* action.participant[=].role.coding.code = #TrainedLayProvider
+* action.action[0].title = "Determine reason for visit"
```

Comment:
There should be some id associated here I think, e.g. HIV.D.1

Reply:
Yeah you're right, I should also be adding relatedAction element which tags the id's 
Thanks @ibacher